lib/eventbrite_sdk/user.rb

```diff
@@ -5,6 +5,7 @@ class User < Resource
     # NOTE: This name is pretty legacy. We should consider renaming
     # to "orders" to normalize things.
     has_many :owned_event_orders, object_class: 'Order', key: :orders
+    has_many :organizations, object_class: 'Organization', key: :organizations
```

Comment:
This looks good but is untested functionality. If you could add tests to cover this we can approve and merge it. You'll find examples of tests for associated relationships in a handful of places like https://github.com/eventbrite/eventbrite-sdk-ruby/blob/master/spec/eventbrite_sdk/organization_spec.rb#L25